packages/xdl/src/detach/AndroidShellApp.js

```diff
@@ -1056,6 +1056,13 @@ export async function runShellAppModificationsAsync(context, sdkVersion, buildMo
     isRunningInUserContext
   );
 
+  // Notifications
+  await regexFileAsync(
+    '"notification_icon_color">#005eff',
```

Comment:
This seems a bit dangerous to me — there's no information in the original manifest that this should be kept in sync with this pattern. Would we be able to write a regex that catches anything between? Like…

```suggestion
    /"notification_icon_color">.*?</s,
```
? (not sure about the `/s` flag, I used https://regex101.com and it allowed for matching tags spanning across many lines. this may be overengineering, but the original comment is still relevant I think)

Reply:
very good point, I'll change this to matching anything in between

We follow the same pattern (relying on the original manifest **not** changing) when it comes to [setting the `splashscreen_background` color as well](https://github.com/expo/expo-cli/blob/master/packages/xdl/src/detach/AndroidShellApp.js#L783-L795), do you think we should make the same changes there? in a separate PR, of course